src/site/topic16.rst

```diff
@@ -46,6 +46,55 @@ Array Iterator
 Linked Iterator
 ---------------
 
+* Similarly, if we want to make an iterator for a collection that has a linked structure for the underlying container, then we make a :download:`LinkedIterator <../src/main/java/LinkedIterator.java>`
```

Comment:
Likewise, this class is only indirectly tested.

Reply:
:+1: